### PR TITLE
docs(support-guidelines): replace slack with discord references

### PR DESCRIPTION
**Pull request by xmfcx**
I've also used https://shields.io/ for the interactive join link.

### CODE REVIEW

**Comment by xmfcx** [comment]
@kenji-miyake I've added `Signed-off-by: M. Fatih Cırıt <mfc@leodrive.ai>` to the commit description but DCO still fails.

File: docs/help/support-guidelines.md [hunk 1]
```diff
@@ -63,11 +63,13 @@ If you can't determine the appropriate repository, ask the maintainers for help
 
 If you want to fix the bug by yourself, discuss the approach with maintainers and submit a pull request.
 
-## Slack
+## Discord
 
-Autoware has a Slack workspace for casual communication between contributors.
+[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)
```
Comment (kenji-miyake) [inline docs/help/support-guidelines.md:68]: Will you add this badge to https://github.com/autowarefoundation/.github/pull/1 as well? I feel it's good.

**Review by kenji-miyake — APPROVED** [review]

File: docs/help/support-guidelines.md (hunk 1 above)
Comment (xmfcx) [reply docs/help/support-guidelines.md:68]: Done! https://github.com/autowarefoundation/.github/pull/1#discussion_r829801076

**Comment by xmfcx** [comment]
@kenji-miyake https://github.com/autowarefoundation/autoware-documentation/runs/5594486445?check_suite_focus=true#step:2:20 this still fails because title is not `fix: replace slack with discord references` why did you change it to be `docs(support-guidelines): replace slack with discord references`? I didn't understand the reason sorry.

> However, since this repository doesn't have any code and it's just a documentation site, we can consider customizing the type.

I understand when the merge is done, it won't care about the title. And it will use the commit message. And it will pass the CI.

But isn't it better to make it consistent with PR titles too?

**Comment by xmfcx** [comment]
Also I don't understand why commit sign-off (DCO) didn't work when I tried but it passes now.

**Comment by xmfcx** [comment]
> Also I don't understand why commit sign-off (DCO) didn't work when I tried but it passes now.

Oh it says:
> Commit sign-off was manually approved.

Why did you need to do that? @kenji-miyake ?

**Comment by kenji-miyake** [comment]
> Why did you need to do that? @kenji-miyake[](https://github.com/xmfcx) ?

@xmfcx It seems the author and committer aren't the same.

image

image

**Comment by kenji-miyake** [comment]
@xmfcx 

> why did you change it to be docs(support-guidelines): replace slack with discord references? I didn't understand the reason sorry.

To follow this type definition.
https://github.com/commitizen/conventional-commit-types/blob/c3a9be4c73e47f2e8197de775f41d981701407fb/index.json#L12

> I understand when the merge is done, it won't care about the title. And it will use the commit message. And it will pass the CI.

I'm sorry, I couldn't understand this part. :thinking: 

1 commit: It won't care about the PR title and use the commit title. It's an undesired behavior. So the CI requires the PR title and the commit title to be consistent.
2+ commits: It will care about the PR title.

> But isn't it better to make it consistent with PR titles too?

Yes, it's better. So I fixed it and now `semantic-pull-request` has passed.

**Comment by xmfcx** [comment]
> @xmfcx It seems the author and committer aren't the same.

Thanks, I was so confused!

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**